<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1077</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="bgwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>800</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1080</width>
     <height>800</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1080</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #0F1035;</string>
   </property>
   <widget class="QPushButton" name="home">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>680</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset;
border-width: 2px;
border-color: #fff;
border-radius: 5px;
background-color: #004AAD;
font: bold 10pt &quot;Times New Roman&quot;;
color: #fff;</string>
    </property>
    <property name="text">
     <string>HOME</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>680</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset;
border-width: 2px;
border-color: #fff;
border-radius: 5px;
background-color: #004AAD;
font: bold 10pt &quot;Times New Roman&quot;;
color: #fff;</string>
    </property>
    <property name="text">
     <string>RESET</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>680</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset;
border-width: 2px;
border-color: #fff;
border-radius: 5px;
background-color: #004AAD;
font: bold 10pt &quot;Times New Roman&quot;;
color: #fff;</string>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>961</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 36pt &quot;Times New Roman&quot;;
color: #fff;


</string>
    </property>
    <property name="text">
     <string>IMAGE PROCESSING ASSISTANT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>741</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>22</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:50px;
color: #fff;
font:  22pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>CHOOSE YOUR IMAGE PROCESSING TOOL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="handwritingbtn">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>270</y>
      <width>180</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: #004AAD;
    border-style: outset;
    border-width: 2px;
    border-radius: 15px;
    border-color: #fff;
    font: bold 24px &quot;Times New Roman&quot;;
    color: #fff;
}</string>
    </property>
    <property name="text">
     <string>Text to
 Handwriting</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>270</y>
      <width>180</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
  background-color: #004AAD;
    border-style: outset;
    border-width: 2px;
    border-radius: 15px;
    border-color: #fff;
    font: bold 24px &quot;Times New Roman&quot;;
    color: #fff;
}</string>
    </property>
    <property name="text">
     <string>Image to Sketch</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>490</y>
      <width>180</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
   background-color: #004AAD;
    border-style: outset;
    border-width: 2px;
    border-radius: 15px;
    border-color: #fff;
    font: bold 24px &quot;Times New Roman&quot;;
    color: #fff;
}</string>
    </property>
    <property name="text">
     <string>Gesture Control</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>490</y>
      <width>180</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: #004AAD;
    border-style: outset;
    border-width: 2px;
    border-radius: 15px;
    border-color: #fff;
    font: bold 24px &quot;Times New Roman&quot;;
    color: #fff;
}</string>
    </property>
    <property name="text">
     <string>Gesture Mouse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>490</y>
      <width>180</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: #004AAD;
    border-style: outset;
    border-width: 2px;
    border-radius: 15px;
    border-color: #fff;
    font: bold 24px &quot;Times New Roman&quot;;
    color: #fff;
}</string>
    </property>
    <property name="text">
     <string>QR READER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>180</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: #004AAD;
    border-style: outset;
    border-width: 2px;
    border-radius: 15px;
    border-color: #fff;
    font: bold 24px &quot;Times New Roman&quot;;
    color: #fff;
}</string>
    </property>
    <property name="text">
     <string>Image to Text</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../image.qrc"/>
 </resources>
 <connections/>
</ui>
